src/App.tsx

```diff
@@ -20,12 +31,15 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={(word) => {
+                  setQuery(word.target.value);
+                }}
```

Comment:
It's better to use event instead of word in the onChange event handler because event is a commonly used and recognized parameter name for event handlers in JavaScript.